web/openapi/openapi_gen.py

```diff
@@ -27,16 +27,14 @@ def generate_example_code(model, grpc_url):
     """
     Generates the GRPC examples codes based on the notes
     """
-    python_code_template = "swagger/templates/python_code.txt"
+    python_code_template = "web/openapi/templates/python_code.txt"
     logging.info(f"Using grpc url:\t{grpc_url}")
     logging.info(f"Using template to create python code:\t\t{python_code_template}")
     environment = Environment(loader=FileSystemLoader("./"))
     template = environment.get_template(python_code_template)
     context = model
     context["url"] = grpc_url
-
-    txt = html.escape(template.render(context)).replace("\n", "<br>")
-    return f"<pre>{txt}</pre>"
+    return template.render(context).replace("\n", "\n        ")
```

Comment:
Just a note to remind someone in the future that this is very brittle, since its whitespace-sensitive.